src/styles/blocks/card.scss

```
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors

```suggestion
  border: 1px solid $c-color;
```